lte/gateway/python/magma/subscriberdb/crypto/milenage.py

```diff
@@ -54,6 +54,35 @@ def generate_eutran_vector(self, key, opc, sqn, plmn):
         kasme = Milenage.generate_kasme(ck, ik, plmn, sqn_bytes, ak)
         return rand, xres, autn, kasme
```

Comment:
I would suggest defining a NamedTuple object containing the 4 variables being returned here, and then returning that object instead of this implicit tuple (you can follow the example here: https://github.com/magma/magma/blob/master/lte/gateway/python/magma/pipelined/gtp_stats_collector.py#L26-L37)

Reply:
I've addressed the comments.

Reply:
I've addressed the comment.